app/controllers/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create 
+    @user = User.new(user_params)
+      if @user.save
+        redirect_to root_url
```

Comment:
Please add flash messages when the user successfully signup in your application. You can provide your own custom message for the user.